apps/connection/src/connection.repository.ts

```diff
@@ -93,7 +93,7 @@ export class ConnectionRepository {
    * @returns Get connection details
    */
   // eslint-disable-next-line camelcase
-  async saveConnectionWebhook(payload: IConnectionInterface): Promise<object> {
+  async saveConnectionWebhook(payload: Connection): Promise<object> {
```

Comment:
Replace object as a return type with appropriate interface.

Reply:
It's not clear what data will come from the webhook, so its return type has been kept as an object. This was decided after a discussion with Ankita Ma'am and you, that for the webhook's APIs, the return type should remain as an object.